curator/base/deploymentconfigs/deploymentconfig.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
```

Comment:
I would prefer to see Deployment resources, which are native to Kubernetes, rather than OpenShift-specific DeploymentConfig resources. Red Hat makes the same recommendation in [their docs on the topic](https://docs.openshift.com/container-platform/4.10/applications/deployments/what-deployments-are.html) ("it is recommended to use Deployment objects unless you need a specific feature or behavior provided by DeploymentConfig objects.").

Reply:
It would also be nice to see a backup job for the curator data, but that can be part of a future pull request.